apps/api/src/config/env-validator.ts

```diff
@@ -8,6 +8,16 @@ const str32 = makeValidator((variable) => {
 
   return variable;
 });
+
+const recommended = makeValidator((variable?: string) => {
+  if (variable == null || variable === '') {
+    // eslint-disable-next-line no-console
+    console.warn(`The env var is optional, but recommended.`);
+  }
+
+  return '';
+});
```

Comment:
This one is WIP just wanted to hear your thoughts about it. I do not think that we make caching mandatory for our users if they don't want to. However, I think that it is a good idea to suggest using it.

Reply:
Will remove it.